### ISSUE TITLE
Why comment out 'netG:evaluate()' in test.lua？

### ISSUE THREAD

**Issue by Naruto-Sasuke** [phillipi/pix2pix#87]
When test the model, shouldn't we use `netG:evaluate()` to make special layers like nn.BatchNormalization() perform differently with that in training?

**Comment by phillipi** [comment]
Actually we found it more effective to have the special layers behave identically between train and test (in retrospect, it seems like that should have been the default all along...). Doing this with batchnorm, with batchsize=1, results in [instance norm](https://arxiv.org/abs/1607.08022). Doing this with dropout is a way to make the network slightly stochastic at test time.
